.readthedocs.yaml

```diff
@@ -0,0 +1,20 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.9"
```

Comment:
Hmm maybe it is the version of python and mkdocs that is the issue.

collapsiblenav build: https://readthedocs.org/projects/civicactions-handbook/builds/18803675/

vs

another PR build: https://readthedocs.org/projects/civicactions-handbook/builds/18782556/